src/audio_core/audio_renderer.cpp

```diff
@@ -16,6 +16,7 @@
 #include "core/settings.h"
 
 namespace {
+static constexpr std::size_t MAX_QUEUED_AUDIO_BUFFERS = 16;
```

Comment:
```suggestion
constexpr std::size_t MAX_QUEUED_AUDIO_BUFFERS = 16;
```

A newline should also be added after this line.